docs/UserGuide.md

```diff
@@ -378,9 +378,52 @@ Format: `delnote INDEX`
 
 Example:
 
-`view 1` shows you the detailed view of  the contact at index 1, then using `delnote 2` will delete the second note in the 
+`view 1` shows you the detailed view of  the contact at index 1, then using `delnote 2` will delete the second note in the
 notes list of the contact
 
+### Add images : `addimg`
+
+Add image(s) to a contact.
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+Take note that images uploaded are duplicated and stored in the data folder of the app.
+If your computer's storage space is a concern for you, please delete the original image
+after you have uploaded it!
+</div>
+
+Format: `addimg INDEX`
+
+![file chooser for images](images/images_file_chooser.png)
+
+- Upon running the command, a file chooser will appear for you to select images from.
+- Images can be in `.png` or `.jpg` formats.
+- Images uploaded cannot have duplicate names.
```

Comment:
When rendered, it doesn't have a difference, but let's standardise using `*` to make lists 😆